src/style.css

```diff
@@ -2,0 +1,109 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url('./fonts/Avenir-Book.ttf') format('truetype');
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 600;
+  src: url('./fonts/Avenir-Heavy.ttf') format('truetype');
+}
+
+body {
+  margin: 0;
+  font-family: Avenir, Arial, sans-serif;
+}
+
+.search__container {
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+}
+
+.search__input {
+  color: #3d4e61;
+  width: 95%;
+  padding-left: 45px;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  font-weight: 300;
+  transition: border-color 0.3s;
+  box-shadow: 0 0 7px rgb(232, 230, 230);
+  font-family: inherit;
+  position: relative;
+  margin-left: 10px;
+  margin-right: 10px;
+}
+
+.search__input::placeholder {
+  color: #3d4e61;
+}
+
+.big {
+  margin-top: 20px;
+  height: 70px;
+  font-size: 16px;
```

Comment:
The classes '.big' and '.small' are fixing the height of the input fields, which could lead to overflow or accidental scroll bars if the content exceeds the height. Consider using padding and line-height to control the size and let the content dictate the height.